models/demos/wormhole/llama31_8b/tt/model_config.py

```diff
@@ -238,6 +243,18 @@ def __init__(self, device, instruct=False):
                 fuse_batch=seq_len <= 2048,
             )
 
+            self.model_config["OUTPUT_MM_PROGCFG"] = ttnn.MatmulMultiCoreReuseMultiCast1DProgramConfig(
+                compute_with_storage_grid_size=(7, 8),
```

Comment:
Refer to the issue# for the workaround maybe.

Reply:
Good catch!
I've added this to all specific program configs to didt issue 👍